packages/flutter_tools/lib/src/ios/fallback_discovery.dart

```diff
@@ -97,37 +82,13 @@ class FallbackDiscovery {
     } on Exception catch (err) {
       _logger.printTrace(err.toString());
     }
-    _logger.printTrace('Failed to connect with log scanning, falling back to mDNS');
+    _logger.printTrace('Failed to connect with log scanning');
     UsageEvent(
       _kEventName,
       'log-failure',
       flutterUsage: _flutterUsage,
     ).send();
 
-    try {
```

Comment:
To make sure I understand, this PR just removes mDNS discovery from fallback discovery, but it will still be used when needed from e.g. flutter attach?

Reply:
Yes, it's still used on `attach`, but no longer used for `drive` or `run`.
https://github.com/flutter/flutter/blob/b8397f69d372a0c2611acfda1435e0a0ba21806a/packages/flutter_tools/lib/src/commands/attach.dart#L263